src/components/headMenu/index.tsx

```diff
@@ -0,0 +1,18 @@
+import { Link } from "react-router-dom";
+import "./headmenu.styles.css";
+
+const HeadMenu = () => {
```

Comment:
HeadMenu?? Its called a Navbar...

Reply:
its header of the page basically I called it as headMenu as in desktop it will be header and on mobile it will be menu

Reply:
This is a poor explanation. Never use this logic infront of a client.

Reply:
Sure Sir